actions/verify-helm-schema/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
       uses: giantswarm/install-binary-action@v1.1.0
       with:
         binary: "schemalint"
-        version: "2.5.0"
+        version: "2.5.1"
```

Comment:
For some reason this did not happen when releasing v2.5.0 [here](https://github.com/giantswarm/schemalint/pull/138/files).